alignak/http/daemon.py

```diff
@@ -95,7 +95,8 @@ def __init__(self, host, port, http_interface, use_ssl, ca_cert,
 
         self.srv = CherryPyWSGIServer((host, port),
                                       cherrypy.Application(http_interface, "/", config),
-                                      numthreads=daemon_thread_pool_size, shutdown_timeout=1)
+                                      numthreads=daemon_thread_pool_size, shutdown_timeout=1,
+                                      request_queue_size=30)
```

Comment:
It modify queue length of cherrypy socket server from 5 to 30 (better for big config)